### PR TITLE
Implement showing drug frequency in `EditMedicineScreen` based on the country

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.clubhouse.io/simpledotorg/story/4782/implement-showing-drug-frequency-in-editmedicinescreen-based-on-the-country

### CODE REVIEW

**Comment by janhavisinghh** [comment]
Added @Honey14 as a reviewer as @vinaysshenoy is on a leave

File: app/src/test/java/org/simple/clinic/drugs/EditMedicineEffectHandlerTest.kt [hunk 1]
```diff
@@ -144,4 +148,15 @@ class EditMedicineEffectHandlerTest {
...
     verifyZeroInteractions(uiActions)
   }
+
+  @Test
+  fun `when load drug frequency choice items effect is received, then load drug frequency choice items`() {
+    // when
+    testCase.dispatch(LoadDrugFrequencyChoiceItems)
+
+    // then
+    val expectedResult = drugFrequencyFactory.provideFields()
```
Comment (Honey14) [inline app/src/test/java/org/simple/clinic/drugs/EditMedicineEffectHandlerTest.kt:158]: We're not testing whether the `drugFrequencyFactory.provideFields()` is returning list of choice items or not. We're just adding the expected list and expecting it to work fine. Can you add this, please?

File: app/src/test/java/org/simple/clinic/drugs/EditMedicineUpdateTest.kt [hunk 2]
```diff
@@ -138,4 +142,33 @@ class EditMedicineUpdateTest {
...
+
+    updateSpec
+        .given(model)
+        .whenEvent(DrugFrequencyChoiceItemsLoaded(DrugFrequencyChoiceItems(drugFrequencyChoiceItems)))
+        .then(assertThatNext(
+            hasModel(model.medicineFrequencyToFrequencyChoiceItemMapLoaded(medicineFrequencyToFrequencyChoiceItemMap)),
+            hasNoEffects()
+        ))
+  }
+
```
Comment (Honey14) [inline app/src/test/java/org/simple/clinic/drugs/EditMedicineUpdateTest.kt:173]: extra space here.

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks great. Left few comments. PTAL?

**Comment by Honey14** [comment]
There are some build failures, waiting for Janhavi to push.

**Comment by janhavisinghh** [comment]
There is some issue happening with jitpack [link](https://jitpack.io/com/github/ben-manes/versions/com.github.ben-manes.versions.gradle.plugin/0.38.0/com.github.ben-manes.versions.gradle.plugin-0.38.0.pom) which is why I'm unable to push currently. Will try again in some time @Honey14

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**